generator/test_case_generator.go

```diff
@@ -122,6 +122,10 @@ var testTypeToTestConfig = map[string][]testConfig{
 		{testDir: "../../../test/restart"},
 		{testDir: "../../../test/acceptance"},
 		{testDir: "../../../test/feature/windows/event_logs"},
+		{
```

Comment:
I don't think we need to run this on all platforms. 
The latest "win-2022" has both a test for ssm and non ssm.
Meaning this will be tested the same 2 times.
This test will only use ssm since it makes the ps1 command simpler.